### DISCUSSION TITLE
sizing `ui.breadcrumbs_card()`

### DISCUSSION THREAD

**Discussion by azim-b** [h2oai/wave#1509] [Q&A] [answered]
Do we plan to support sizing the `ui.breadcrumbs_card()`, for example like we do for `ui.persona()` (xs, s, m...) or for other components (ex: height/width)?

The problem I am facing is that the breadcrumbs are too large, especially when compared to the heading, for example a `ui.section_card()` in my case.

Example screenshot:
![image](https://user-images.githubusercontent.com/64787868/174459716-1f85d712-50e2-455d-9076-9539132365e1.png)

Workaround in mind, haven't tried, implement a breadcrumbs_card like `ui.inline()` component using `ui.link()`

**Comment by mturoci** [comment, marked as answer]
>Do we plan to support sizing the ui.breadcrumbs_card()

Probably not. The reason is that there is a very limited number of usecases and possible placement positions (I can think about just 1 - like in your screenshot), so a single unified style should be one-size-fits-all.

> The problem I am facing is that the breadcrumbs are too large, especially when compared to the heading

Note that this is a section heading, not a page heading. Page heading should be the most prominent item on a page, while section heading is usually around the same size/smaller than regular text, it's just extra info. The hierarchy in general is:

1. Page heading
2. Breadcrumbs
3. Rest of the text

However, we might want to add support for page heading card - feel free to file a feature request.

**Reply by azim-b** [reply]
ok - this makes sense. What's the best way to create a page heading today?

**Reply by mturoci** [reply]
The only way today is probably using text_xl at the top of the card like you do in the pic (`My ui.table()`), but when it comes to having multiple cards, then there is not much IIRC hence the need for a new card.

**Reply by azim-b** [reply]
got it - thanks
